cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/CallSubqueryTest.scala

```diff
@@ -126,23 +135,59 @@ class CallSubqueryTest extends DocumentingTest {
         )))) { resultTable() }
     }
 
-    section("Aggregation and side-effects", "subquery-aggregation") {
-      p("""Subqueries can be useful to do aggregations for each row and to isolate side-effects.
-          #This example query creates five `Clone` nodes for each existing person.
-          #The aggregation ensures that cardinality is not changed by the subquery.
-          #Without this, the result would be five times as many rows.""".stripMargin('#'))
+    section("Cardinality and aggregation", "subquery-aggregation") {
+      p(
+        """Returning subqueries change the cardinality of the query: The results of each subquery call are collected and together they give the result of the
+          |enclosing query.""".stripMargin)
```

Comment:
Shouldn't we rather say that they are the result of the `CALL` clause
```suggestion
          |`CALL` clause.""".stripMargin)
```

Reply:
Agreed. I would then also suggest to replace call with "invocation" in the line before to not use the word "call" twice with two different meanings.